frontend/dist/css/index.css

```diff
@@ -0,0 +1,125 @@
+html,
+body {
+  color: black;
+  font-family: 'RobotoDraft', 'Roboto', sans-serif;
+  font-size: 14px;
+  margin: 0 auto;
+  width: 100%;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url('https://char.gd/assets/_featured/pull_left-1.jpg');
+}
+
+.card-panel {
+  margin: 150px auto;
+}
+
+.card-container {
+  display: flex;
+  display: -webkit-flex; /* Safari */
+  display: -ms-flexbox; /* IE */
+  align-items: center;
+  flex-direction: column; 
+  justify-content: center;
+}
+
+.github {
+  background-color: rgb(23, 21, 22);
+  color: white;
+}
+
+.linkedin {
+  background-color: #004471;
+  color: white;
+}
+
+nav {
+  height: 80px;
+}
+
+.navbar-fixed {
+  height: 80px;
+}
+
+.news-img {
+  margin-top: 1em;
```

Comment:
Can you double check if using em is safe here? Usually, when you use px and em together something gets messed up.